packages/db/package.json

```diff
@@ -1,8 +1,15 @@
 {
   "name": "@astrojs/db",
   "version": "0.10.3",
-  "description": "",
+  "description": "Add SQLite and Astro Studio support to your Astro site.",
```

Comment:
I think we can mention LibSQL over SQLite specifically here, like we do in the docs?

Reply:
Yep, +1